src/main/java/net/minecraftforge/fmp/capabilities/ICapabilityWrapper.java

```diff
@@ -0,0 +1,19 @@
+package net.minecraftforge.fmp.capabilities;
+
+import java.util.Collection;
+
+import net.minecraftforge.common.capabilities.Capability;
+
+/**
+ * Interface that allows mods to provide an implementation of a capability in the case that there's more than one implementation
+ * for one side of a block (multiparts).
+ * 
+ * @see Capability
+ * @see CapabilityWrapperRegistry
+ */
+public interface ICapabilityWrapper<T>
+{
```

Comment:
It could be merged into the capability system, yeah. That should be just fine. As I told you on IRC, that's what's in charge of wrapping multiple implementations of a cap into just one so the TE can return it. I'll move it to Forge's package.